### PR TITLE
Homework 03 complete

### PR DESCRIPTION
**Pull request by cinck**
Please check my homework for Lesson #3

### CODE REVIEW

File: homework03/tasks.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+# 1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази.
+print("Task 1:")
+print("I love Python\n"*42)
```
Comment (acman) [inline homework03/tasks.py]: nit: better to use spaces before and after operations
```suggestion
print("I love Python\n" * 42)
```

File: homework03/tasks.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# 2. Створити змінну age_in_month, надавши їй значення вашого поточного віку в місяцях.
+print("\nTask 2:")
+age_in_month = 451
+print(age_in_month)
+
+# 3. Створити змінну age_in_years, в яку записати ваш вік в роках на основі попередньої змінної.
+print("\nTask 3:")
+age_in_years = age_in_month//12
```
Comment (acman) [inline homework03/tasks.py]:
```suggestion
age_in_years = age_in_month // 12
```

File: homework03/tasks.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+print(age_in_years)
+
+# 4. Створити змінну my_age, яка буде містити рядок “Му name is … I’m … years old”,
+#   де на замість пропусків буде підставлятись ваші імʼя та вік.
+#   Значення віку слід брати зі змінної age_in_years.
+print("\nTask 4:")
+name = "Oleksandr"
+
+my_age = f"My name is {name}. I'm {age_in_years} years old."
```
Comment (acman) [inline homework03/tasks.py:22]: 👍

File: homework03/tasks.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+print(f"Is my number == \'1\' ?  -{new_var == '1'}")
+print(f"Is my number > 2 ?  -{new_var > 2}")
+print(f"Is my number < 5 ?  -{new_var < 5}")
+print(f"Is my number != 1 ?  -{new_var != 1}")
+print(f"Is my number == 1 ?  -{new_var == 1}")
+
+# 6. Створити змінні a=2, b=5, c=6. На основі цих змінних створити змінну d,
+#   значення якої має бути “256”
+print("\nTask 6:")
+a, b, c = 2, 5, 6
```
Comment (acman) [inline homework03/tasks.py:40]: 👍

File: homework03/tasks.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+d = int(str(a)+str(b)+str(c))
```
Comment (acman) [inline homework03/tasks.py]:
```suggestion
d = int(str(a) + str(b) + str(c))
```

**Review by acman — APPROVED** [review]
Nice job here! Only leave several comments about code style

**Comment by cinck** [comment]
Thank you.

**Comment by cinck** [comment]
Could you please clarify what do you mean " leave several comments about code style"?
What kind of comments should it be?

**Pull request merged**